src/sidebar/services/test/session-test.js

```diff
@@ -170,32 +162,28 @@ describe('SessionService', () => {
         });
       });
 
-      it('should cache the returned profile data', () => {
+      it('should cache the returned profile data', async () => {
         const session = createService();
-        return session
-          .load()
-          .then(() => {
-            return session.load();
-          })
-          .then(() => {
-            assert.calledOnce(fakeApi.profile.read);
-          });
+        await session.load();
+        await session.load();
```

Comment:
This is unrelated, but I think it is clearer: two consecutive `load` operations triggers the reading of the profile only once.